README.adoc

```diff
@@ -466,6 +466,76 @@ endif::[]
 // ===============================================================================================
 
 == Testing the application
+
+You can test your application manually, but automated tests ensure code quality because they trigger a failure whenever a code change introduces a defect. In this section, you'll create unit tests for the gRPC server service and integration tests for the `query` service.
+
+=== Implementing unit tests for the gRPC server service
+
+// file 0
+system/pom.xml
+[source, xml, linenums, indent=0, role="code_column hide_tags=copyright"]
+----
+include::finish/system/pom.xml[]
+----
+
+The required dependencies are already added to the `pom.xml` Maven configuration file for you, including JUnit5, [hotspot=grpc-testing file=0]`grpc-testing`, and [hotspot=mockito-core file=0]`mockito-core` libraries. The `grpc-testing` dependency provides utilities for testing gRPC services and is used for creating a mock gRPC server for simulating client-server communication during testing. The `mockito-core` dependency is needed to enable the use of Mockito.
+
+// file 1
+// Create SystemServiceTest.java
+[role="code_command hotspot file=1", subs="quotes"]
+----
+#Create the `SystemServiceTest` class.#
+`system/src/test/java/it/io/openliberty/guides/system/SystemTest.java`
+----
+
+SystemServiceTest.java
+[source, Java, linenums, indent=0, role="code_column hide_tags=copyright"]
+----
+include::finish/system/src/test/java/io/openliberty/guides/system/SystemServiceTest.java[]
+----
+
+// file 2
+SystemService.java
+[source, Java, linenums, indent=0, role="code_column hide_tags=copyright"]
+----
+include::finish/system/src/main/java/io/openliberty/guides/system/SystemService.java[]
```

Comment:
The file name should be SystemServiceTest.java and not SystemTest.java; see: http://lg-dev1.fyre.ibm.com:4020/guides/grpc-intro.html#implementing-unit-tests-for-the-grpc-server-service
The test clss also had compilation errors